firefox-ios/firefox-ios-tests/Tests/ClientTests/Utils/RemoteSettingsUtilTests.swift

```diff
@@ -0,0 +1,34 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+@testable import Client
+import XCTest
+import MozillaAppServices
+
+// TODO: Replace direct remote settings with the mock
+class RemoteSettingsUtilTests: XCTestCase {
```

Comment:
These tests will change, I added one for the boiler plate code however once I know which remote settings we will be targeting (Ex. Search / Shavar etc.) it will get updated.

So in theory this can also be ignored although I might leverage this for further testing here in this PR